module/synchronization/core_rapid_test.go

```diff
@@ -32,8 +33,8 @@ func populatedBlockStore(t *rapid.T) []flow.Header {
 type rapidSync struct {
 	store          []flow.Header
 	core           *Core
-	idRequests     map[flow.Identifier]int // pushdown automaton to track ID requests
-	heightRequests map[uint64]int          // pushdown automaton to track height requests
```

Comment:
Why don't we just use map to bool? Maybe I'm missing it but I don't see anywhere we increment this value that would cause it to ever be something greater than 1?

Reply:
Or, did you mean to increment the value on each request and decrement it on each response?

Reply:
> Or, did you mean to increment the value on each request and decrement it on each response? 

Yep, that's the idea.